docs/DeveloperGuide.md

```diff
@@ -2,21 +2,57 @@
 layout: page
 title: Developer Guide
 ---
-* Table of Contents
-{:toc}
+
+## Table of Contents
```

Comment:
I'm not a fan of changing the table of contents to be not automatically-generated, especially in a long document like this one.